CONTRIBUTING.md

```diff
@@ -61,9 +61,48 @@ If you want to verify the changelog entries as described by a range of Git revis
 
     ./unreleased.sh master..
 
+## Writing a good changelog entry
+
+Writing good changelog entries is **important**: as a developer, it gives visibility on your contribution; as a user, it makes clear what is new, what changes and two deal with those, making the product more accessible and your work more meaningful.
+
+The raw changelog is used to compile a meaningful summary of changes across releases. This happens some time after the PR has been merged and the person taking the responsability of summarizing new user-facing features must be in the position of easily understand the nature of the change and report it. The ideal changelog entry can be more or less incorporated verbatim in the release notes.
+
+Here are a few practical tips
+
+* the first term to appear should be the affected component -- [here's a list](#list-of-components-for-changelog-entries)
+* write as many as necessary changelog entries as necessary
+* don't be _too_ succinct: a single entry does **not have to** fit on a single line
+* on the other end, if the size grows beyond 5-6 lines, rather add a link to a relevant documentation or issue with more details
+* the ultimate target are end users: focus on the impact on them, tell them what's new or how to deal with a change
+
+### List of components for changelog entries
```

Comment:
Maybe add a note that this is not necessarily an exhaustive list. It’s totally fine to use something else or spell it slightly differently (e.g. DAML Assistant or SDK Assistant or somtehing like that)  it just needs to be obvious which component it belongs to but there is no need to check with this list.

Reply:
Totally agree the list is not exhaustive, but if your change does happen to fit in one of the listed component, I don't see a reason to spell it differently.

Reply:
I would like this to be one. If it's not, someone should raise this so that we can keep this list up-to-date for as much as possible.

Reply:
See my comment and suggestion below.